source/includes/openstack/_securityGroups.md

```diff
@@ -90,6 +95,7 @@ Create a security group in an environment.
 ```shell
 curl -X DELETE \
     -H "MC-Api-Key: your_api_key"
```

Comment:
Looks like I forgot to add a `\` in my previous PR - can you add one when you remove the space below?